HACKING.md

```diff
@@ -42,6 +42,46 @@ to configure it for this repository.
 Tests can be run using `make test`, which will run all test forms. Specific types of
 tests can be run with other testing targets shown in `make help`.
 
+In addition to unit tests in `tests/unit`, which can be run with `make test-units`, a
+number of integrated tests in `tests/spread` can be run with
+[Spread](https://github.com/snapcore/spread). See the [general
+notes](https://github.com/snapcore/snapcraft/blob/main/TESTING.md#spread-tests-for-the-snapcraft-snap)
+and heed these Rockcraft-specific instructions:
+
+Initialize and update Git submodules to fetch Spread-related helper scripts with:
+
+```bash
+git submodule init
+git submodule update
+```
+
+Spread needs the Rockcraft snap. Duplicate it by running this at the root of the
+repository:
+
+```bash
+snapcraft --use-lxd
```

Comment:
```suggestion
snapcraft pack
```

Current best practice to pack core24 snaps is to spell out the command

Reply:
Good catch, thanks. b0733f8